<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-18L6O1V" failures="1" tests="3" name="Surefire test" time="0.04" errors="0" timestamp="2024-10-12T23:59:46 IST">
  <testcase classname="TestCases" name="testAdd" time="0.005"/>
  <testcase classname="TestCases" name="testFailedAdd" time="0.006">
    <failure type="java.lang.AssertionError" message="expected [9] but found [15]">
      <![CDATA[java.lang.AssertionError: expected [9] but found [15]
at TestCases.testFailedAdd(TestCases.java:24)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 39 stack frames]]>
    </failure>
  </testcase> <!-- testFailedAdd -->
  <testcase classname="TestCases" name="testSub" time="0.0"/>
</testsuite> <!-- Surefire test -->
